packages/server/dataloader/customLoaderMakers.ts

```diff
@@ -150,13 +149,15 @@ export const userTasks = (parent: RethinkDataLoader) => {
           const {first, after, userIds, teamIds, archived} = key
           const dbAfter = after ? new Date(after) : r.maxval
 
+          let teamTaskPartial = r.table('Task').getAll(r.args(teamIds), {index: 'teamId'})
+          if (userIds) {
+            teamTaskPartial = teamTaskPartial.filter((row) => r(userIds).contains(row('userId')))
```

Comment:
in the case of the team archive, userIds === null because we want all the tasks for team members past & present, so it's cheapest to simply not filter on userId